app/src/sass/crn_app/_elements-wrappers.scss

```diff
@@ -78,6 +78,7 @@ body {
 
 a {
     color: $primary;
+    font-weight: 600;
```

Comment:
This seems to change the font-weight for all of the analysis views. Is that intentional?